src/xcode/ENA/ENA/Source/Views/Common/HtmlTextView.swift

```diff
@@ -47,7 +47,9 @@ class HtmlTextView: UITextView {
 			NSAttributedString.Key.foregroundColor: UIColor.enaColor(for: .textTint)
 		]
 
+		isUserInteractionEnabled = true
 		isEditable = false
+		dataDetectorTypes = [.link, .phoneNumber]
```

Comment:
This text view is also used in Privacy screen in Onboarding. This change shouldn't break anything, but if you didn't I would still check that are links are still working there.

Reply:
Jumping out of the onboarding flow was strongly discouraged during design discussions.

Reply:
Now it works, but tapping on an email there crashes the app. I will open a new discussion for that.

Reply:
I agree with @EisvogelSAP , I will inform the PO about this.

Reply:
Has this been aligned with PO?

Reply:
@EisvogelSAP The PO said to follow the issue. In the issue, only the AppInformation screens are supposed to be interactive (so NOT the onboarding stuff). But this should be already handled by me moving this piece of code outside of the HTMLTextView.